src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+
+### What is web3? {#what-is-web3}
+
+Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchain, cryptocurrency, and NFTs to give power back to the users in the form of ownership. Chris Dixon said it best: web1 was read-only, web2 is read/write, web3 will be read/write/own.
+
+#### Core ideas of web3 {#core-ideas}
+
+Although it's challenging to provide a rigid definition of is and is not web3, a few core principles guide its creation.
+
+- **Web3 is decentralized:** instead of large swathes of the internet controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
+- **Web3 is permissionless:** everyone has equal access to participate in web3, and no one gets excluded.
+- **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the infrastructure banks and payment processors.
+- **Web3 is trustless:** it operates using incentives and mechanisms instead of relying on trusted third-parties
+
+### Why is web 3.0 important? {#why-is-web3-important}
+
+So why does web3 matter so much? Web3's killer features aren't isolated and don't fit into neat categories. But, for simplicity, we've tried to separate them to make them easier to understand.
+
+#### Ownership {#ownership}
+
+Web3 gives you ownership of your digital assets in an unprecedented way. For example, say you're playing a web2 game. If you purchase an in-game item, it is tied directly to your account. If the game creators delete your account, you will lose these items. Or, if you stop playing the game, you lose the value you invested into your in-game items.
+
+Web3 allows for direct ownership through [non-fungible tokens (NFTs)](/nft/). No one, even the game's creators, has the power to take away your ownership. And, if you stop playing, you can sell your in-game items and recoup their value.
+
+<ButtonLink to="/nft/">
+  More on NFTs
+</ButtonLink>
+
+#### Governance {#governance}
+
+As a Web2 platform increases in popularity, the power of the individual user's opinion decreases. Simultaneously, the power of the platform's governing body wields also increases. Over time, this power imbalance can spiral out of control until the platform can ignore grievances with no repercussions.
+
+Web3 lets users participate in the decision making of a platform through tokens. A recent example of this is Ethereum Name Service (ENS), which distributed 25% of its voting rights to existing users by sending ENS tokens to their wallets. Anyone with ENS tokens can vote on proposals about the future of ENS. ENS is a decentralized autonomous organization (DAO) because its ownership gets distributed amongst its token holders.
```

Comment:
```suggestion
Web3 empowers users to participate in the decision making of a platform through tokens. A recent example of this is Ethereum Name Service (ENS), which distributed 25% of its voting rights to existing users by sending ENS tokens to their wallets. Anyone with ENS tokens can vote on proposals about the future of ENS. ENS is a decentralized autonomous organization (DAO) because its ownership gets distributed amongst its token holders.
```